venues/ICLR.cc/2019/Conference/python/iclr19.py

```diff
@@ -286,6 +286,36 @@
 )
 
 
+# User "registration".
```

Comment:
yes, exactly. this was the simplest way I could think to do it. do you have another suggestion?

Reply:
I'm not convinced about the solution, we are adding more notes that we don't need. And the scripts is getting more complex. 

I wouldn't worry too much, when we enable the bids they will see the active venues.

Reply:
Also they are invited to complete a questionnaire, right?

Reply:
Reviewers are invited to complete the questionnaire, not ACs. Reviewers who did not respond to the questionnaire won't see ICLR in their active venues until they do.

the bid invitation won't trigger this either, because it does not meet either of the requirements in home.js

Reply:
Let me check the home logic. It should display venues where you have submitted a paper or you have a completed or pending task.